src/network/networkd.c

```diff
@@ -67,6 +67,9 @@ int main(int argc, char *argv[]) {
         if (r < 0)
                 log_warning_errno(r, "Could not create runtime directory 'lldp': %m");
 
+        /* Create a directory for the generated transient network services */
+        mkdir_p("/run/systemd/network", 0755);
```

Comment:
This should either look at the error code, or have a (void) prefix to make Coverity happy.